### DISCUSSION TITLE
Tina missing some files?

### DISCUSSION THREAD

**Discussion by peterhoang** [danielcgilibert/blog-template#60] [Q&A]
I'm new to the entire tech stack so I'm having trouble figuring out how to get Tina working. The Readme implies that it should work out of the box, but I think it's missing some files.

**Comment by PetteriHeino** [comment]
I was able to get Tina running with a help of copilot. I am trying to run it on my macbook and URL Local    http://localhost:4321/ works, but http://localhost:4001/graphql gives me error Error: Body must be a string. Received: undefined."

Copilot says it has something to do with config.ts where clinetId and Token are null, but then again I found out that in local mode those can be null.
